### PR TITLE
Use not protected branch to store signatures

### PR DESCRIPTION
**Pull request by andreigaevsky**
Do not delete this branch.

### CODE REVIEW

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -19,5 +19,5 @@ jobs:
         with:
           path-to-signatures: 'signatures/cla.json'
           path-to-cla-document: 'https://github.com/exadel-inc/face-recognition-service/blob/develop/CLA.md'
```
Comment (pospielov) [inline .github/workflows/cla.yml:21]: Please update path to CLA

File: .github/workflows/cla.yml [hunk 2]
```diff
@@ -19,5 +19,5 @@ jobs:
...
-          branch: 'develop'
+          branch: 'CLA-signatures'
           allowlist: Elijas,daffy-,sbvalijon,stashess,kemerava,mmisachenka,isobolev,endypanda,mjakowiec,dmitry-boiko,pospielov,andreigaevsky,ivandodz,aabramovich-dev,DmitryRomaniuk,omorgunova-pl,github-actions
```
Comment (pospielov) [inline .github/workflows/cla.yml:23]: Is it a good idea to remove all people from this list? So they have to approve CLA using proper procedure?

File: .github/workflows/cla.yml (hunk 1 above)
Comment (andreigaevsky) [reply .github/workflows/cla.yml:21]: We will update CLA.md through develop branch, won't we? So there always will be the actual version.

File: .github/workflows/cla.yml (hunk 2 above)
Comment (andreigaevsky) [reply .github/workflows/cla.yml:23]: I thought this procedure for outside Exadel contributors only, but as we have rapidly changing staff, it's a good idea to remove all.

Comment (pospielov) [reply .github/workflows/cla.yml:23]: no, this procedure for all contributors, as we use personal github accounts

**Review by pospielov — APPROVED** [review]

**Pull request merged**